tools/cockpit.spec

```diff
@@ -364,8 +361,6 @@ troubleshooting, interactive command-line sessions, and more.
 Summary: Cockpit bridge server-side component
 Requires: glib-networking
 Provides: cockpit-ssh = %{version}-%{release}
-# cockpit-ssh moved from dashboard to bridge in 171
-Conflicts: cockpit-dashboard < 170.x
```

Comment:
This is unrelated, but ok -- we don't support any RHEL or other distro any more with such an old cockpit.